xc7/yosys/synth.tcl

```diff
@@ -1,22 +1,18 @@
+# Import the commands from the plugins to the tcl interpreter
 yosys -import
 
 plugin -i xdc
 plugin -i fasm
-#Import the commands from the plugins to the tcl interpreter
-yosys -import
```

Comment:
Isn't this required?

Reply:
But it's done in the first line of the script.

Reply:
If you read the comment, the `yosys -import` imports new commands that come from the XDC and FASM plugins.

Reply:
Ahh..., you're right.